### PR TITLE
Added supported features by remote servers

### PR DESCRIPTION
**Pull request by Bugagazavr**
Some information about supported servers and features

### CODE REVIEW

File: content/setup/overview.md [hunk 1]
```diff
@@ -7,6 +7,19 @@ menu = "installation"
 toc = true
 +++
 
+# Features support
+
+| Feature/Remote            | GitHub  | GitLab  | BitBucket           | Gogs    |
+|---------------------------|---------|---------|---------------------|---------|
+| Supported version         | --/--   | 8.2+    | --/--               | --/--   |
+| VCS                       | **git** | **git** | **git ( only )**    | **git** |
```
Comment (tboerger) [inline content/setup/overview.md]: I think for BitBucket hg should also work.

File: content/setup/overview.md [hunk 2]
```diff
@@ -7,6 +7,19 @@ menu = "installation"
...
+| Auth method               | ouath2  | oauth2  | oauth               | manual  |
+| Push events               | **yes** | **yes** | **yes**             | **yes** |
+| Push tags events          | **yes** | **yes** | **yes**             | no      |
+| Merge requests            | **yes** | **yes** | **public ( only )** | no      |
+| Commit statuses           | **yes** | **yes** | **yes**             | no      |
+| Restrict by organizations | **yes** | no      | **yes**             | no      |
```
Comment (tboerger) [inline content/setup/overview.md]: Is this broken for GitLab? There have been an orgs option...

Comment (Bugagazavr) [inline content/setup/overview.md]: Yep, currently this is not works, but i can add support for this

File: content/setup/overview.md (hunk 1 above)
Comment (Bugagazavr) [inline content/setup/overview.md]: @bradrydzewski or @gtaylor can you verify this?

File: content/setup/overview.md (hunk 2 above)
Comment (tboerger) [inline content/setup/overview.md]: Should be good to have it, than it's on the same level as GitHub.

File: content/setup/overview.md (hunk 1 above)
Comment (gtaylor) [inline content/setup/overview.md]: I don't use mercurial, but it was my understanding that we only supported git for Bitbucket as well. I see that we have a mercurial plugin, but the webhook values are a bit different for Hg repos. Not wildly different, but different enough to where we wouldn't automatically have working Hg support just by virtue of having an Hg plugin.

**Comment by gtaylor** [comment]
So I've been wanting to create a separate page just for walking people through the process of selecting and configuring a remote. This would be a great thing to have for that, with each remote getting its own sub-page.

For now, let's merge this in since it's a missing piece that helps. I may get this sub-page worked out as soon as tonight, so look for it to move there if I can figure this out in a way that I'm happy with.

LGTM

File: content/setup/overview.md (hunk 1 above)
Comment (bradrydzewski) [inline content/setup/overview.md]: we have a plugin for mercurial that should work with Bitbucket mercurial repositories, however, this isn't something I've thoroughly tested:

```
clone:
  image: hg
```

**Comment by bradrydzewski** [comment]
Just wondering if this should be the first thing we display on the install page? This seems like useful information, but maybe not the most important thing to lead with

**Comment by gtaylor** [comment]
@bradrydzewski No, but see my paragraph on having a separate pages for remotes. This is something I want to do in the next day. In the meantime, the info is shown (we could even shuffle it down two sections), even if it's in a funky spot.

**Pull request merged**